Documentation/Helm-Charts/operator-chart.md

```diff
@@ -70,6 +70,9 @@ The following table lists the configurable parameters of the rook-operator chart
 | `csi.csiCephFSPluginVolume` | The volume of the CephCSI CephFS plugin DaemonSet | `nil` |
 | `csi.csiCephFSPluginVolumeMount` | The volume mounts of the CephCSI CephFS plugin DaemonSet | `nil` |
 | `csi.csiCephFSProvisionerResource` | CEPH CSI CephFS provisioner resource requirement list | see values.yaml |
+| `csi.csiLeaderElectionLeaseDuration` | Duration in seconds that non-leader candidates will wait to force acquire leadership. | `137s` |
+| `csi.csiLeaderElectionRenewDeadline` | Deadline in seconds that the acting leader will retry refreshing leadership before giving up. | `107s` |
+| `csi.csiLeaderElectionRetryPeriod` | Retry period in seconds the LeaderElector clients should wait between tries of actions. | `26s` |
```

Comment:
```suggestion
| `csi.csiLeaderElectionLeaseDuration` | Duration in seconds that non-leader candidates will wait to force acquire leadership. | `137s` |
| `csi.csiLeaderElectionRenewDeadline` | Deadline in seconds that the acting leader will retry refreshing leadership before giving up. | `107s` |
| `csi.csiLeaderElectionRetryPeriod` | Retry period in seconds the LeaderElector clients should wait between tries of actions. | `26s` |
```

how these values are taken? On this doc I see values are quite lesshttps://github.com/kubernetes-csi/external-provisioner?tab=readme-ov-file#command-line-options

Reply:
@subhamkrai These are the values suggested by the openshift storage team long back, btw what the suggested change I cannot see any.

Reply:
```text
--leader-election-lease-duration <duration>: Duration, in seconds, that non-leader candidates will wait to force acquire leadership. Defaults to 15 seconds.

--leader-election-renew-deadline <duration>: Duration, in seconds, that the acting leader will retry refreshing leadership before giving up. Defaults to 10 seconds.

--leader-election-retry-period <duration>: Duration, in seconds, the LeaderElector clients should wait between tries of actions. Defaults to 5 seconds.```

Reply:
The defaults value are 10s,10s,5s repectively

Reply:
@subhamkrai i dont have any plan to change the default values what we have in Rook from very long time and its working fine without any issue. https://github.com/rook/rook/blob/b9fa0f44d5dd00bdbcc99097338674a1fd55815f/pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml#L34-L36 these are the same values we have release 1.10. This PR is exposing user options to change the values if they want and doesn't change any existing values

Reply:
Okay, got it.